tasks/selinux_load_module.yml

```diff
@@ -27,7 +27,9 @@
       become: false
 
     - name: Install module
-      when: not module_file.stat.checksum in checksum
+      when: >-
+        (not module_file.stat.exists) or
+        (module_file.stat.exists and not module_file.stat.checksum in checksum)
```

Comment:
while this seems correct, unfortunately it will break https://github.com/linux-system-roles/selinux/blob/main/tests/tests_selinux_modules.yml#L149 because an error won't be raised by this code.  I think we should not fix this.  Or, if we do, we need to figure out if it is ok to always raise an error if `not module_file.stat.exists`